demisto_sdk/commands/find_dependencies/find_dependencies.py

```diff
@@ -966,6 +969,52 @@ def _find_pack_dependencies(pack_id, id_set, verbose_file, exclude_ignored_depen
 
         return pack_dependencies
 
+    @staticmethod
+    def build_all_dependencies_graph(pack_ids, id_set, verbose_file, exclude_ignored_dependencies=True):
```

Comment:
add type hints

Reply:
I see that the standard in the file is to specify the types in the docstring so that's what I did, do you want me to add type hints to all methods in this file instead?

Reply:
yes please, so mypy can run on it